packages/terra-table/tests/wdio/single-select-table-spec.js

```diff
@@ -1,73 +1,80 @@
-const viewports = Terra.viewports('medium');
-
-describe('Single-Select Table', () => {
+Terra.describeViewports('Single-Select Table', ['medium'], () => {
   describe('Single-Select Table - Interactions', () => {
     before(() => {
       browser.url('/#/raw/tests/terra-table/table/single-select-table/selectable-table');
       browser.waitForVisible('table');
     });
 
-    // reset focusable point for keyboard navigation
-    beforeEach(() => browser.click('[data-terra-table-header-cell]:nth-child(1)'));
-
     describe('Single-Select Table - Click Operations', () => {
       it('selects the second row', () => {
         browser.click('[class*="row"]:nth-child(1)');
       });
 
-      Terra.it.matchesScreenshot('first-row', { viewports });
+      Terra.it.validatesElement('first-row');
 
       it('selects the second row', () => {
         browser.click('[class*="row"]:nth-child(2)');
       });
 
-      Terra.it.matchesScreenshot('second-row', { viewports });
+      Terra.it.matchesScreenshot('second-row');
 
       it('selects the third row', () => {
         browser.click('[class*="row"]:nth-child(3)');
       });
 
-      Terra.it.matchesScreenshot('third-row', { viewports });
+      Terra.it.matchesScreenshot('third-row');
     });
 
     describe('Single-Select Table - Keyboard ENTER key', () => {
+      // reset focusable point for keyboard navigation
+      before(() => {
+        browser.click('[data-terra-table-header-cell]:nth-child(1)');
+        browser.pause(50);
+      });
+
       it('selects the first row', () => {
         browser.keys(['Tab', 'Enter']);
       });
 
-      Terra.it.matchesScreenshot('first-row', { viewports });
+      Terra.it.validatesElement('first-row');
 
       it('selects the second row', () => {
         browser.keys(['Tab', 'Enter']);
       });
 
-      Terra.it.matchesScreenshot('second-row', { viewports });
+      Terra.it.matchesScreenshot('second-row');
 
       it('selects the third row', () => {
         browser.keys(['Tab', 'Enter']);
       });
 
-      Terra.it.matchesScreenshot('third-row', { viewports });
+      Terra.it.matchesScreenshot('third-row');
     });
 
     describe('Single-Select Table - Keyboard SPACE key', () => {
+      // reset focusable point for keyboard navigation
+      before(() => {
+        browser.click('[data-terra-table-header-cell]:nth-child(1)');
+        browser.pause(50);
```

Comment:
For some reason removing the `beforeEach` hook caused the blue line that for some reason appears at the bottom of the 1st and 2nd row selected screenshots to sometimes not appear, this causes it to always appear.